src/assets/styles/base.css

```diff
@@ -0,0 +1,155 @@
+:root {
+  --root-font-size: 16px;
+
+  --global-layout-size: 80rem; /* 1280px */
+  --global-layout-gutter: 2rem;
+
+  --global-color-green-400: hsl(160, 100%, 37%);
+  --global-color-green-a20: hsla(160, 100%, 37%, 0.2);
+  --global-color-neutral-100: hsl(0, 0%, 10%);
+  --global-color-neutral-500: hsl(0, 0%, 50%);
+  --global-color-neutral-900: hsl(0, 0%, 94%);
+  --global-color-neutral-1000: hsl(0, 0%, 100%);
+
+  --global-animation-fade-up: fade-up 0.5s ease-in-out;
+  --global-animation-fade-down: fade-down 0.5s ease-in-out;
+
+  --global-text-color: var(--global-color-neutral-1000);
+  --global-text-color-hover: var(--global-color-neutral-500);
+  --global-background-color: var(--global-color-neutral-100);
+
+  color-scheme: light dark;
+
+  @media (prefers-color-scheme: light) {
+    --global-color-text: var(--global-color-neutral-100);
+    --global-color-text-hover: var(--global-color-neutral-500);
+    --global-background-color: var(--global-color-neutral-1000);
+  }
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  font-weight: normal;
+}
+
+html {
+  font-size: var(--root-font-size);
+  min-height: 100svh; /* svh is more reliable than vh on mobile */
+  color: var(--global-text-color);
+  background: var(--global-background-color);
+  scroll-behavior: smooth;
+  transition:
+    color 0.5s,
+    background-color 0.5s;
+}
+
+body {
+  font-family:
+    Inter,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Nunito',
+    sans-serif;
+  font-size: var(--root-font-size);
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+header {
+  position: sticky;
+  top: 0;
+  left: 0;
+  display: flex;
+  width: 100%;
+  /* You can break up padding/margin shorthand */
+  padding-inline: 0; /* This is the same as padding-left and padding-right */
+  padding-block: 4rem; /* This is the same as padding-top and padding-bottom */
+  /* Shorthands will usually be more concise, also note how this "cascades" (overwrites) the previous properties */
+  padding: 0;
+}
+
+main {
+  margin: 0 auto;
+  padding: var(--global-layout-gutter);
+  padding-top: calc(var(--global-layout-gutter) * 3);
+  max-width: var(--global-layout-size);
+}
+
+a,
+.green {
+  padding: 0.25rem;
+  text-decoration: none;
+  color: var(--global-color-green-400);
+  transition: 0.4s;
+
+  &:hover {
+    background-color: var(--global-color-green-a20);
+  }
+}
+
+h1 {
+  margin-bottom: var(--global-layout-gutter);
+  font-size: 2.8rem;
+  font-weight: 700;
+  animation: var(--global-animation-fade-up);
+}
+
+h2 {
+  margin-block: var(--global-layout-gutter) calc(var(--global-layout-gutter) / 2);
+  font-size: 1.4rem;
+  font-weight: 600;
+  animation: var(--global-animation-fade-up);
+}
+
+.organization-example {
+  /* Some people prefer to organize css properties alphabetically */
```

Comment:
@kingloomy I was wondering about how people like to organize them... Do you have a recommendation for a linter that has property organization? Just using Prettier at the moment. But something like this: https://blog.logrocket.com/using-stylelint-improve-lint-css-scss-sass/